drivers/disk/usdhc.c

```diff
@@ -2534,20 +2534,22 @@ static int usdhc_sd_init(struct usdhc_priv *priv)
 		usdhc_set_bus_width(base, USDHC_DATA_BUS_WIDTH_4BIT);
```

Comment:
Please change commit title to "drivers: usdhc: fix sdcard spec 1.0 support"
Please fix commit body and describe the issue or the fix. spec 3.0/spec 1.10 is probably SD card specification?

Reply:
Yes, It is Physical Layer Simplified Specification Version.
I have some SD cards with spec 1.0 that can’t work normally under Zephyr. The reason is that the CMD6 operation is performed on it in usdhc, butstart to  spec 1.10 and above support CMD6, which will cause usdhc_sd_initfailure. This patch can let The SD Card of spec 1.0 can work normally.
Since SD Card has multiple spec versions, CMD 6 Group 3 (driver strength and current limit) is only supported after spec 3.0, so it also needs to be restricted.